evaluation_experiments/EXP3_cibersort_eval.ipynb

```diff
@@ -0,0 +1,6160 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        plt.title(f'Correlation of Real and BayesPrism Pred Props, {num} Cell Missing')
</pre><p>this should say CIBERSORTx not BayesPrism in the plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pred_missing_celltypes/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
nice catch